src/applications/vaos/new-appointment/components/ConfirmationPage/ConfirmationDirectScheduleInfo.jsx

```diff
@@ -51,32 +51,26 @@ export default function ConfirmationDirectScheduleInfo({
 
         <div className="vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
           <div className="vads-u-flex--1 vads-u-margin-top--2 vads-u-margin-right--1 vaos-u-word-break--break-word">
-            <dl className="vads-u-margin--0">
-              <dt className="vads-u-font-weight--bold">
-                {clinic?.serviceName}
-              </dt>
-              <dd>
-                {!!facilityDetails && (
-                  <>
-                    {facilityDetails.name}
-                    <br />
-                    <FacilityAddress facility={facilityDetails} />
-                  </>
-                )}
-              </dd>
-            </dl>
+            <h4 className="vads-u-font-size--h4">{clinic?.serviceName}</h4>
```

Comment:
Looks like this should be an `h3`, since the header before this is an h2. Not sure about the class, would need to look at the review page to make sure the text size is correct.

Reply:
Done.